File: 프로그래머스/unrated/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        int answer = 0;
        int[] antDmg = {5,3,1};

        for (int i = 0; i < antDmg.length; i++) {
            answer += hp/antDmg[i];
            hp %= antDmg[i];
        }
        return answer;
    }
}

File: 프로그래머스/unrated/120837. 개미 군단/개미 군단.js
function solution(hp) {
    let answer = 0;
    const antDmg = [5, 3, 1];

    for (let i = 0; i < antDmg.length; i++) {
        answer += Math.floor(hp / antDmg[i]);
        hp %= antDmg[i];
    }
    return answer;
}